convert-coverage.py

```diff
@@ -0,0 +1,13 @@
+"""Convert a cov.xml report so that it is usable locally"""
```

Comment:
Which tools can read cov.xml after running this script?

Reply:
[This](https://github.com/ryanluker/vscode-coverage-gutters) is the one I'm utilizing, but I believe it to be a standard format that many tools understand.